test/strings/strcmp.c

```diff
@@ -0,0 +1,14 @@
+#include <string.h>
+#include "smack.h"
+
+// @expect verified
+
+int main(void) {
+  char *smaller = "AAAA";
+  char *bigger = "aaaa";
+
+  int comparison = strcmp(smaller,bigger);
+
```

Comment:
No need for this new line. Like I said, make your files tighter in terms of new lines.